3.3-nginx/conf.d/default.conf

```diff
@@ -0,0 +1,14 @@
+server {
+    listen       ${PORT};
+    server_name  ${SERVERNAME};
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+}
```

Comment:
Please, configure your IDE properly.

In Codium > Preferences > Settings search for "final" and look for

- [x]  Render Final Newline
- [x]  Insert Final Newline
- [x]  Trim Final Newlines